portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase

```diff
@@ -20,6 +20,30 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-195389"
+	@priority = 3
+	test AssertHighlightingSearchResultsWithAdvancedSyntaxIsEnabled {
+		Portlet.gotoPortletOptions(
+			portletName = "Search Bar",
+			portletOption = "Configuration");
+
+		SelectFrame(locator1 = "IFrame#CONFIGURATION");
+
+		Click.clickAtNotVisible(
+			checkboxName = "Use Advanced Search Syntax",
+			locator1 = "Checkbox#ANY_CHECKBOX");
+
+		IFrame.closeFrame();
```

Comment:
Use `SearchPortlet#configureEmbeddedSearchBar`. You'll need to create a new conditional for advanced search.